src/utils.ts

```diff
@@ -78,21 +78,16 @@ const networksIds = Object.keys(networks);
 const mainnetNetworkIds = Object.keys(networks).filter(
   (id) => !networks[id].testnet
 );
-const testnetNetworkIds = Object.keys(networks).filter(
-  (id) => networks[id].testnet
-);
 
 ajv.addKeyword({
   keyword: 'snapshotNetwork',
-  validate: function (schema, data) {
-    // @ts-ignore
-    const snapshotEnv = this.snapshotEnv || 'default';
+  validate: function (schema, data, options) {
+    const snapshotEnv = options?.snapshotEnv || 'default';
```

Comment:
I tried many things previously but there is no option to pass `options` here, this is called internally in agv, so I was using `.call` in line `435`
```suggestion
  validate: function (schema, data) {
    // @ts-ignore
    const snapshotEnv = this.snapshotEnv || 'default';
```